src/Account/Auth.js

```diff
@@ -77,10 +77,14 @@ class AccountAuth {
     });
   }
 
-  exchangeIdServiceCode(strategyFlags) {
-    return this.launcher.http.sendGet(`${ENDPOINT.LOGIN_FRONTEND}/api/exchange`, null, null, true, {
```

Comment:
Unofficial Fortnite Client for Node.js
npm version
npm downloads
license
paypal

This library supports only fortnite client. If you need launcher, check node-epicgames-client

Installation